app/src/openxr/cpp/OpenXRExtensions.cpp

```diff
@@ -47,6 +47,7 @@ void OpenXRExtensions::Initialize() {
 #elif PICOXR
     // Pico incorrectly advertises this extension as supported but it makes Wolvic not work.
     sSupportedExtensions.erase(XR_EXTX_OVERLAY_EXTENSION_NAME);
+    sSupportedExtensions.erase(XR_EXT_HAND_INTERACTION_EXTENSION_NAME);
```

Comment:
Add a comment explaining that 5.11 added it but that there is a bug in the runtime that prevents other interaction profiles from being used (like the ones for controllers) if that is active

Reply:
Done